src/overlays/actors/ovl_Boss_Fd/z_boss_fd.c

```diff
@@ -9,7 +16,84 @@ void BossFd_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BossFd_Update(Actor* thisx, GlobalContext* globalCtx);
 void BossFd_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_808CB6A8(BossFd* this, GlobalContext* globalCtx); // Setup flying
+void func_808CB718(BossFd* this, GlobalContext* globalCtx); // Flying
+void func_808CDE30(BossFd* this, GlobalContext* globalCtx); // Wait
+void func_808CE01C(BossFd* this, GlobalContext* globalCtx); // Environment
```

Comment:
Some prototypes like this one arent needed because the function bodies are declared before being used